### PR TITLE
[NUI] BaseComponent: Add RiveAnimationView Component.

### PR DESCRIPTION
**Pull request by taehyub**
This reverts commit bbbe71cc1f8c47516933be9851482564e023a895.

### Description of Change ###

    [NUI] BaseComponent: Add RiveAnimationView Component.
    
    RiveAnimationView shows and control the rive animations which are vector graphics based animation.
    Also RiveAnimationView allows the user interaction and user customizing development
    such as runtime animation change, runtime user interaction and runtime vector property change.
    
    Classes
     Tizen.NUI.BaseComponents.RiveAnimationView
    
    Dependency
     dali-csharp-binder
        https://review.tizen.org/gerrit/#/c/platform/core/uifw/dali-csharp-binder/+/261323/
        https://review.tizen.org/gerrit/#/c/platform/core/uifw/dali-csharp-binder/+/261832/
     dali-extension
        https://review.tizen.org/gerrit/#/c/platform/core/uifw/dali-extension/+/261321/
     rive-tizen: https://review.tizen.org/gerrit/#/admin/projects/platform/core/uifw/rive-tizen
    
    NUI Example
     test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/RiveAnimationTest.cs

### API Changes ###

 - ACR:

### CODE REVIEW

File: src/Tizen.NUI/src/internal/Interop/Interop.RiveAnimationView.cs [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+{
+    internal static partial class Interop
+    {
+        internal static partial class RiveAnimationView
+        {
+            [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_RiveAnimationView_New__SWIG_0")]
+            public static extern global::System.IntPtr New(string jarg1);
+
+            [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_RiveAnimationView_EnableAnimation")]
+            public static extern void EnableAnimation(global::System.Runtime.InteropServices.HandleRef jarg1, string jarg2, bool jarg3);
```
Comment (dongsug-song) [inline src/Tizen.NUI/src/internal/Interop/Interop.RiveAnimationView.cs:28]: please change the names of parameters (ex: `jarg1`, `jarg2`) into meaningful names for the next patch~

File: src/Tizen.NUI/src/public/BaseComponents/RiveAnimationView.cs [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+        public void EnableAnimation(string animationName, bool on)
+        {
+            Interop.RiveAnimationView.EnableAnimation(SwigCPtr, animationName, on);
+        }
+
+        /// <summary>
+        /// Play animation.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public void PlayAnimation()
```
Comment (dongsug-song) [inline src/Tizen.NUI/src/public/BaseComponents/RiveAnimationView.cs:65]: How about change the name as just `Play()` ?
for example, `LottieAnmationView` and `Animation` classes have the names of `Play()`, `Pause()` and `Resume()`.

File: src/Tizen.NUI/src/internal/Interop/Interop.RiveAnimationView.cs (hunk 1 above)
Comment (taehyub) [reply src/Tizen.NUI/src/internal/Interop/Interop.RiveAnimationView.cs:28]: I will change the name of parameters at the next patch.

**Review by dongsug-song — APPROVED** [review]
looks good to me.
the newly added APIs are all hidden-APIs, so it is OK this time.

**Pull request merged**